### PR TITLE
update sm68 matrix, add via support

### PR DESCRIPTION
**Pull request by hineybush**
## Description

Fix matrix misalignment and add VIA support

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
Is this a "v2" with a different matrix, and there are sm68 boards in the wild with the previous mapping?

File: keyboards/hineybush/sm68/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+       KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,               KC_TRNS,
+       KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,                                     KC_TRNS, KC_TRNS, KC_TRNS,               KC_TRNS, KC_TRNS, KC_TRNS
+    )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    return true;
+}
+
+
```
Comment (fauxpark) [inline keyboards/hineybush/sm68/keymaps/via/keymap.c:53]:
```suggestion
```

**Comment by hineybush** [comment]
@zvecr no, same as the original. I typo'd the original matrix and had bottom row keys one over than they should have been. None are in the wild yet.

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**